### DISCUSSION TITLE
Migrating from v1

### DISCUSSION THREAD

**Discussion by YuriOku** [cosmo-sims/MUSIC2#3] [Q&A] [answered]
I have a question regarding transitioning from version 1 to 2. I compared initial conditions generated by MUSIC v1 and v2 with the same config file, and they are more inconsistent than the machine precision. Is this expected? And is there an option to get a consistent result in v2? I think this information is helpful for those migrating from v1. I'm sorry if I missed the description.

Below is the config file I used:
```
[setup]
boxlength               = 60
zstart                  = 100
levelmin                = 7
levelmin_TF             = 9
levelmax                = 10

padding                 = 16  
overlap                 = 4
baryons                 = no  
use_2LPT                = no
use_LLA                 = no  

#Ellipsoidal refinement region defined on unity cube
region                     = ellipsoid
region_ellipsoid_matrix[0] = 822.07,  -86.84, -661.59
region_ellipsoid_matrix[1] = -86.84,  673.36, 243.17
region_ellipsoid_matrix[2] = -661.59, 243.17, 1295.95
region_ellipsoid_center    = 0.527531, 0.522087, 0.449525


[cosmology]
Omega_m                 = 0.272
Omega_L                 = 0.728
Omega_b                 = 0.0455
H0                              = 70.2
sigma_8                 = 0.807
nspec                   = 0.961
transfer                = eisenstein

# only for v2 to set Omega_nu = 0
m_nu1 = 0
m_nu2 = 0
m_nu3 = 0

[random]
cubesize                = 256
seed[8]                 = 95064
seed[9]                 = 31415
seed[10]                = 27183

[poisson]
fft_fine                = yes
accuracy                = 1e-4
grad_order      = 6
laplace_order   = 6


[output]
format			= gadget2
filename		= ics_gadget.dat
```

**Comment by ohahn** [comment, marked as answer]
Yes, the change that allowed the major reduction in memory footprint unfortunately required that. The difference is at the level of the composition of the white noise at different levels. If compatibility is needed, then one currently has to stick to either v1 or v2. However, v2 now is also compatible with PANPHASIA white noise fields. I will clarify this better in the main README...

**Reply by YuriOku** [reply]
Thank you for the clarification!
